.github/actions/format/index.js

```diff
@@ -0,0 +1,54 @@
+const core = require("@actions/core")
+const fs = require("fs")
+const yaml = require("js-yaml")
+
+try {
+  const yamlFiles = fs.readdirSync("./").filter(file => file.endsWith(".yaml"))
+  format(yamlFiles)
+} catch (error) {
+  core.setFailed(error.message)
+}
+
+function format(yamlFiles) {
+  yamlFiles.forEach(file => {
+    const content = fs.readFileSync(file)
+
+    let data = yaml.load(content)
+    data = consistent(data)
+    data = sort(data)
+
+    const formattedContent = yaml.dump(data, {quotingType: '"', lineWidth: -1})
+    fs.writeFileSync(file, formattedContent)
+  })
+}
+
+function consistent(data) { // one-record subdomains might not be an array
+  for (const key in data) {
+    if (!Array.isArray(data[key])) {
+      const array = [data[key]]
+      data[key] = array
+    }
+  }
+
+  return data
+}
```

Comment:
This additionally will directly modify the actual records themselves, which I am against any automation modifying data contents

Reply:
This won't change the record, it just handles cases like this:

```
www:
  ttl: 1
  type: A
```

vs the more consistent

```
www:
- ttl: 1
  type: A
```